debezium-core/src/main/java/io/debezium/util/SchemaNameAdjuster.java

```diff
@@ -160,16 +160,18 @@ public static SchemaNameAdjuster create(Logger logger) {
      * @return the validator; never null
      */
     public static SchemaNameAdjuster create(Logger logger, ReplacementOccurred uponConflict) {
+        final Logger adjusterLogger = LoggerFactory.getLogger(logger.getName() + "." + SchemaNameAdjuster.class.getSimpleName());
```

Comment:
Admittedly not a fan of dynamically obtaining loggers; do we actually need to pass in the logger from the caller?

Reply:
Right now we allow to configure te schemanameandjuster per connector type. The question is different - do w want to remove this capbailty form the user or not?

Reply:
Hum, hum, I see. But then at the very least we'd have to document this, otherwise no one will know about that special adjuster logger name.

Reply:
@gunnarmorling Docs added